cclib/parser/orcaparser.py

```diff
@@ -2181,23 +2187,33 @@ def energy_intensity(line):
         # Magnitude (a.u.)       :      0.00000
         # Magnitude (Debye)      :      0.00000
         #
-        if line.strip() == "DIPOLE MOMENT":
+        # TODO: add quadrupole moment parsing, which can be optionally calculated with ORCA
+
+        # the origin/reference might be printed in multiple places in the output file
+        # depending on the calculation type
+        if line.startswith("The origin for moment calculation is"):
+            tmp_reference = line.split()[-3:]
+            reference_x = float(tmp_reference[0].replace("(", "").replace(",", ""))
```

Comment:
```suggestion
            reference_x = float(tmp_reference[0][1:-1])
```

Reply:
Happy with using slices instead, but think there is a slight bug with that change at the moment. Turns out there is an extra space between the parenthesis, which I didn't see
```
The origin for moment calculation is the CENTER OF MASS  = ( 3.220560,  4.367513 -0.707204)
```
It should be `[:-1]` instead because now it reads `3.220560,` as `0.22056`

Reply:
Not worth the potential pain of what the output might look at if the COM has more than one digit to the left of the decimal, so this is fine.